server/data/mappers/inPrisonEducationMapper.test.ts

```diff
@@ -1,139 +1,172 @@
 import moment from 'moment/moment'
 import type { InPrisonEducation } from 'viewModels'
-import toInPrisonEducation from './inPrisonEducationMapper'
+import { toInPrisonEducation, toCourseStatus } from './inPrisonEducationMapper'
 import {
   aValidEnglishLearnerEducation,
   aValidMathsLearnerEducation,
 } from '../../testsupport/learnerEducationTestDataBuilder'
 
 describe('inPrisonEducationMapper', () => {
-  it('should map a Curious LearnerEducation to an InPrisonEducation view model given LearnerEducation does not have an actualEndDate value', () => {
-    // Given
-    const apiLearnerEducation = { ...aValidEnglishLearnerEducation() }
-    apiLearnerEducation.learningActualEndDate = null
-
-    const expectedInPrisonEducation: InPrisonEducation = {
-      prisonId: 'MDI',
-      prisonName: 'MOORLAND (HMP & YOI)',
-      courseCode: '008ENGL06',
-      courseName: 'GCSE English',
-      courseStartDate: moment('2021-06-01').toDate(),
-      courseCompleted: false, // expect courseCompleted to be false because there is no value for learningActualEndDate
-      courseCompletionDate: null,
-      isAccredited: false,
-      grade: null,
-      source: 'CURIOUS',
-    }
-
-    // When
-    const actual = toInPrisonEducation(apiLearnerEducation)
-
-    // Then
-    expect(actual).toEqual(expectedInPrisonEducation)
+  describe('toInPrisonEducation', () => {
```

Comment:
This looks like a lot of change, but its actually just indentation because I've wrapped the existing tests in a `describe('toInPrisonEducation', () => {` block to separate them from the new tests I've written below

Reply:
I wish whitespace changes were hidden by default, without having to select that option and reload the page (or adding `&w=1` to the URL).